ambry-frontend/src/main/java/com/github/ambry/frontend/NamedBlobPutHandler.java

```diff
@@ -145,48 +145,6 @@ private void start() {
       }
     }
 
-    /**
```

Comment:
Is there any logic change in this class or only refactoring?

Reply:
Just refactoring